core-common/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -34,6 +34,7 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.CLASS)
 public @interface Mapper {
+    String DEFAULT_IMPL_NAME = "*Impl";
```

Comment:
Doh, sorry, GitHub masked some characters. I meant something like "&lt;CLASS_NAME&gt;" and "&lt;PACKAGE_NAME&gt;".